### PR TITLE
test generate client file

### PR DESCRIPTION
**Pull request by aniketmaurya**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/agreed via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

Add test for `LitServer.generate_client_file`

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in GitHub issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/test_lit_server.py [hunk 1]
```diff
@@ -440,3 +440,11 @@ def test_middlewares_inputs():
 
     with pytest.raises(ValueError, match="middlewares must be a list of tuples"):
         ls.LitServer(ls.test_examples.SimpleLitAPI(), middlewares=(RequestIdMiddleware, {"length": 5}))
+
+
+def test_generate_client_file(tmp_path, monkeypatch):
```
Comment (Andrei-Aksionov) [inline tests/test_lit_server.py:445]: Is the goal to check that the url is correct, or you might also want to check the whole content of the generated file?

Comment (aniketmaurya) [reply tests/test_lit_server.py:445]: nice point! when I was writing the test, my goal was to only test if the port is correctly updated in the template.

Comment (aniketmaurya) [reply tests/test_lit_server.py:445]: testing whole content now

**Review by ethanwharris — APPROVED** [review]

**Comment by fcakyon** [comment]
Hey @aniketmaurya, it's generating the port incorrectly for me. I'm using 0.2.3. Is it a known bug?

**Comment by aniketmaurya** [comment]
> Hey @aniketmaurya, it's generating the port incorrectly for me. I'm using 0.2.3. Is it a known bug?

this PR was merged after the release. Can you try using the main branch @fcakyon ?

**Pull request merged**